cordova-plugin-firebase.opam

```
@@ -10,6 +10,7 @@ synopsis: "Binding to the firebase cordova plugin using gen_js_api"
 build: [[ "dune" "build" "-j" jobs "-p" name "@install" ]]
 depends: [
   "ocaml" { >= "4.08.0" }
+  "dune" {>= "2.7"}
```

I would say that it is wiser to have:

```suggestion
  "dune" {>= "2.8"}
```

since we have this:

https://github.com/besport/ocaml-cordova-plugin-firebase/blob/3eaddb1cc6a1e493ea77ccfb7d816b07662d4519/dune-project#L1

---

I will have to redo all my "dune" references in my others library then.

---

Alright!

---

It should be good now, I've updated all the "dune" references of my libraries to "2.8".